### ISSUE TITLE
Bug: class names not extracted correctly from class attribute

### ISSUE THREAD

**Issue by fiznool** [prettier/plugin-pug#72]
## Info

| Tool     | Version       |
| -------- | ------------- |
| Plugin   | v1.1.6        |
| Prettier | v1.19.1       |
| Node     | v12.15.0       |
| OS       | mac |

## Input

```pug
div(class="bg-red-100 border-l-4 border-red-600 text-red-700 p-4 mb-4")
```

## Output or Error

```pug
.bg-red-100(.border-l-4.border-red-600.text-red-700.p-4.mb-4)
```

## Expected Output

```pug
.bg-red-100.border-l-4.border-red-600.text-red-700.p-4.mb-4
```

## Additional Context

As you can see, everything but the first class is wrapped inside parenthesis, which causes the class names not to be applied.

**Comment by Shinigami92** [comment]
Thanks, I thought I had written tests for this case 🤔
I had an internal refactor and maybe this influenced this
Anyway, fixed in 1.1.7

https://github.com/prettier/plugin-pug/commit/7bbc4236e29fef7ae2809adf23a1ba0c4ebe1883

**Comment by fiznool** [comment]
Thanks!
